source/module_io/write_Vxc.hpp

```diff
@@ -1,9 +1,12 @@
+#pragma once
```

Comment:
in which condition will this line of code take effect?

Reply:
To avoid redefination when the header is included for many times, equivalent to:
```
#ifndef XXX_H  
#define XXX_H
 code
#endif
```